src/main/java/edu/oregonstate/cs361/battleship/BattleshipModel.java

```diff
@@ -159,10 +159,6 @@ public void hardplace(){
         hardgenerate("computer_submarine");
         hardgenerate("computer_aircraftC");
```

Comment:
Also just noticed missing the AI() for aircraft carrier. The function isn't even there